libs/agentc_cli/agentc_cli/cmds/index.py

```diff
@@ -91,24 +88,26 @@ def logging_printer(content: str, *args, **kwargs):
     else:
         printer = click.secho
 
-    printer(DASHES, fg=KIND_COLORS[kind])
-    printer(kind.upper(), bold=True, fg=KIND_COLORS[kind])
-    printer(DASHES, fg=KIND_COLORS[kind])
-    next_catalog = index_catalog(
-        embedding_model,
-        meta_version,
-        version,
-        get_path_version,
-        kind,
-        catalog_path,
-        source_dirs,
-        scan_directory_opts=DEFAULT_SCAN_DIRECTORY_OPTS,
-        printer=printer,
-        print_progress=True,
-        max_errs=DEFAULT_MAX_ERRS,
-    )
-
-    if not dry_run:
-        next_catalog.dump(catalog_path)
-        click.secho("\nCatalog successfully indexed!", fg="green")
-    click.secho(DASHES, fg=KIND_COLORS[kind])
+    for kind in kinds:
+        # TODO: The kind needs a security check as it's part of the path?
```

Comment:
`kind` can only be of type `tool` or `prompt` right? if yes, then the users cant input any other string so it seems safe

Reply:
👍 (assertion now added earlier in function)